Rakefile

```diff
@@ -20,14 +20,23 @@ Bundler::GemHelper.install_tasks
 
 require "rake/testtask"
 
-Rake::TestTask.new(:test) do |t|
-  t.libs = ["lib", "test"]
-  t.ruby_opts = ["-rsetup"]
-  t.pattern = "test/**/*_test.rb"
-  t.verbose = false
-
-  # Hide warnings emitted by our dependencies.
-  t.warning = false
+namespace :test do
+  ["postgresql", "mysql2"].each do |adapter|
+    adapter_short = adapter[/^[a-z]+/]
```

Comment:
Given it's not simple to run the test suite against multiple database, I suggest we remove the distinction between `mysql` and `mysql2` by using the adapter name (`mysql2`) - this way we'll remove code mapping from one to the other thus reducing the overall cognitive overhead.